utils/zerovec/src/map/kv.rs

```diff
@@ -89,3 +89,20 @@ impl<'a> ZeroMapKV<'a> for String {
         f(g)
     }
 }
+
+impl<'a> ZeroMapKV<'a> for Vec<u8> {
```

Comment:
thought: this can be on `Vec<T: ULE>`, yes?

Reply:
(also `ZeroVec<'static, T: AsULE>`)

Reply:
Since ZV is in flux, I'm going to merge this, with an action item to track this comment: https://github.com/unicode-org/icu4x/issues/676#issuecomment-924586831